Podcast/FetchFeedEndpointRequest.swift

```diff
@@ -11,34 +11,22 @@ import SwiftyJSON
 
 class FetchFeedEndpointRequest: EndpointRequest {
     
-    var offset: Int
-    var max: Int
+    var pageSize: Int
```

Comment:
This will give us errors in FeedViewController. We need to also be fixing this View Controller before we push this.